tests/models/test_models.py

```diff
@@ -181,6 +186,119 @@ def test_scvi(save_path):
     )
     assert "lr-Adam" in m.history.keys()
 
+    # test non-default mmd parameters and validate mmd is tracked in model history
+    model = SCVI(adata, n_latent=n_latent, mmd_mode="fast", mmd_loss_weight=10.0)
+    model.train(max_epochs=1, check_val_every_n_epoch=1, train_size=0.5)
+    assert model.is_trained is True
+    assert model.module.mmd_loss_weight == 10.0
+    assert model.module.mmd_mode == "fast"
+    z = model.get_latent_representation()
+    assert z.shape == (adata.shape[0], n_latent)
+    assert "mmd_loss_train" in model.history.keys()
+    assert "mmd_loss_validation" in model.history.keys()
+    assert len(model.history["mmd_loss_train"]) == 1
+    assert len(model.history["mmd_loss_validation"]) == 1
+    assert not np.isnan(model.history["mmd_loss_train"].values[0][0])
+    assert not np.isnan(model.history["mmd_loss_validation"].values[0][0])
+    model.get_mmd_loss()
```

Comment:
From what I understand, this call does not look like it is actually doing validation. I followed the same pattern as in the rest of this test (for example see get_elbo, a bit further above), but I am curious how these calls (such as get_elbo, get_marginal_ll, etc.) actually perform validation (if they do).